components/core/tests/fixtures/plan.sh

```diff
@@ -8,11 +8,12 @@ pkg_deps=()
 pkg_build_deps=()
 
 do_build() {
-  cp -v $PLAN_CONTEXT/signme.dat signme.dat
+  cp -v "$PLAN_CONTEXT/signme.dat" signme.dat
 }
 
+# shellcheck disable=SC2154
```

Comment:
Is `shellcheck` complaining that `do_install` isn't used?

Reply:
It's complaining that `pkg_prefix` is referenced but not assigned. This is a general issue we have with all the special hab variables that are set outside the plan.

Reply:
ah, OK